client/App.tsx

```diff
@@ -1,7 +1,135 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import Dashboard from './components/Dashboard';
+import Signup from './components/Signup';
+import Login from './components/Login';
+import '../public/styles.css';
+
+export interface ISState {
+    style: {
+        loginBox: string;
+        signupBox: string;
+        dashboard: string;
+    };
+}
+
+// const GITHUB_CLIENT_ID = 'fef59d869191d88d4bc9';
+// const gitHubRedirectURL = 'http://localhost:3000/api/auth/github';
+// const path = '/';
 
 function App() {
-    return <div className="App">hey</div>;
+    const [style, setStyle] = useState<ISState['style']>({
+        loginBox: 'none',
+        signupBox: 'block',
+        dashboard: 'none',
```

Comment:
The Dashboard option is just a temporary one, since I needed to design it and implement chart, fetching component, etc... It will be implemented in React Routes with public and private routes.

We can use the logic above for switching between signing up and logging in if you would like.